### PR TITLE
torch 2.2.0

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

Update supported versions of PyTorch

### Related tickets

133202

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -385,7 +385,7 @@ conda install -c conda-forge nncf
 - Ubuntu\* 18.04 or later (64-bit)
 - Python\* 3.7 or later
 - Supported frameworks:
-  - PyTorch\* >=2.0, <2.2
+  - PyTorch\* >=2.1, <2.3
   - TensorFlow\* >=2.8.4, <=2.12.1
   - ONNX\* ~=1.13.1
   - OpenVINO\* >=2022.3.0
```
Comment (alexsu52) [inline README.md:391]: This repository is tested on Python* 3.8.10, PyTorch* **2.2.0** (NVidia CUDA\* Toolkit 11.8) and TensorFlow* 2.12.1 (NVidia CUDA\* Toolkit 11.8).

Comment (AlexanderDokuchaev) [reply README.md:391]: done

**Comment by AlexanderDokuchaev** [comment]
nightly/job/TriggerBetta/611/ - OK
post_training_quantization/289/ - OK
nightly/job/test_examples/249/ - OK
nightly/job/torch_nightly/135/ - OK
torch_weekly/22/ - OK

File: nncf/version.py [hunk 2]
```diff
@@ -11,6 +11,6 @@
 
 __version__ = "2.9.0"
 
-BKC_TORCH_VERSION = "2.1.2"
+BKC_TORCH_VERSION = "2.2.0"
 BKC_TORCHVISION_VERSION = "0.16.2"
```
Comment (KodiaqQ) [inline nncf/version.py:15]: Torchvision update missed?

**Review by KodiaqQ — COMMENTED** [review]
Do not forget to update **e2e** for the 2.1 version (if needed) as well.

**Review by alexsu52 — APPROVED** [review]
LGTM

File: nncf/version.py (hunk 2 above)
Comment (AlexanderDokuchaev) [reply nncf/version.py]: fixed

**Review by KodiaqQ — APPROVED** [review]

**Pull request merged**